### PR TITLE
[DOCS] Improve DataGrip Document

### PR DESCRIPTION
**Pull request by zwangsheng**
zwangsheng [Closes #428](https://github.com/yaooqinn/kyuubi/pull/428)    Target Issue Doc [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### _Why are the changes needed?_

fix quick start with DataGrip Document
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Review by yaooqinn — APPROVED** [review]

File: docs/quick_start/quick_start_with_datagrip.md [hunk 1]
```diff
@@ -1,41 +1,40 @@
...
-## 5.2 Preparation
-### 5.2.1 Get DataGrip And Install
+## Preparation
+### Get DataGrip And Install
 Please go to [Download DataGrip](https://www.jetbrains.com/datagrip/download) to get and install an appropriate version for yourself.
-### 5.2.2 Get Kyuubi Started
+### Get Kyuubi Started
 [Get kyuubi server started](https://kyuubi.readthedocs.io/en/latest/quick_start/quick_start.html) before you try DataGrip with kyuubi.
 
 For debugging purpose, you can use ```tail -f``` or ```tailf``` to track the server log.
```
Comment (pan3793) [inline docs/quick_start/quick_start_with_datagrip.md:11]: suggest use single ` for inline code

File: docs/quick_start/quick_start_with_datagrip.md [hunk 2]
```diff
@@ -1,41 +1,40 @@
...
 
 ![select database](../imgs/dataGrip/select_database.png)
-### 5.3.3 Datasource Driver
+### Datasource Driver
 You should first download the missing driver files. Just click on the link below, DataGrip will download and install those. 
 
 ![datasource and driver](../imgs/dataGrip/datasource_and_driver.png)
-### 5.3.4 Generic JDBC Connection Settings
+### Generic JDBC Connection Settings
 After install drivers, you should configure the right host and port which you can find in kyuubi server log. By default, we use ```localhost``` and ```10009``` to configure.
```
Comment (pan3793) [inline docs/quick_start/quick_start_with_datagrip.md:24]: `

**Comment by pan3793** [comment]
One tip @zwangsheng use `[x]` instead of `[√ ]` for checkbox

**Comment by zwangsheng** [comment]
> Almost looks good! Thanks @zwangsheng

ok， I will fix what said above

**Review by pan3793 — APPROVED** [review]
LGTM, waiting CI pass, and I will merge it! Thanks.

**Comment by zwangsheng** [comment]
> LGTM, waiting CI pass, and I will merge it! Thanks.

ok,thanks

**Pull request closed without merging**